php/ubiquity/.ubiquity/config.php

```diff
@@ -0,0 +1,2 @@
+<?php
+return array("host"=>"127.0.0.1","port"=>8090,"sessionName"=>null);
```

Comment:
what is this configuration for ?

Reply:
Hi @waghanza 
this is for the embedded php server:
is used to execute:
`Ubiquity serve`
But irrelevant here